backend/curriculum_tracking/admin.py

```diff
@@ -164,6 +163,14 @@ class UserAdmin(BaseUserAdmin):
         "groups",
         # "user_permissions",
     )
+    list_per_page = 25
+
+    actions = ["bulk_deactivate_users"]
+
+    def bulk_deactivate_users(modeladmin, request, users):
```

Comment:
This would be better:

```
def bulk_deactivate_users(self, request, users):
    users.update(active=False)
```

I changed modeladmin to self, it looks like an instance method so it should be self.